content/monitors/downtimes.md

```diff
@@ -55,12 +55,24 @@ To schedule downtime, click the "Schedule Downtime" button in the upper right.
 
 2. Set a schedule.
   {{< img src="monitors/downtimes/downtime-schedule.png" alt="downtime-schedule" responsive="true" style="width:80%;">}}
-  You can set a start date and time or leave the field empty to immediately start the downtime. You may also set a repeating schedule to accommodate regularly scheduled downtimes.
+  You can set a start date and time or leave the field empty to immediately start the downtime. You may also set a [repeating schedule to accommodate regularly scheduled downtimes](#recurring-downtimes).
 
 3. Add an optional message to notify your team
   {{< img src="monitors/downtimes/downtime-notify.png" alt="downtime-notify" responsive="true" style="width:80%;">}}
   Enter a message to notify your team about this downtime. The message field allows standard [markdown formatting][5] as well as Datadog's @-notification syntax. The *Notify your team* field allows you to specify team members or send the message to a service [integration][6].
 
+## Recurring Downtimes
+
+Recurring downtimes allow you to create a downtime that is started and stopped and a recurring interval or pattern. 
+
+These are useful if you have for example a regularly scheduled maintenance window, non-peak or peak hours for which you have configured a different monitor to account for this change in behavior and want to suppress an otherwise noisy monitor.
+
+The downtimes will be scheduled and cancelled at the specified constraints, by cancelling the existing downtime and creating a new downtime with the same constraints associated to the user which configured this downtime.
+
```

Comment:
I'm not clear what this section is referring to

Reply:
yeah, re-wording this.